src/defineFunction.js

```diff
@@ -0,0 +1,122 @@
+import functions from "./functions";
+import {groupTypes as htmlGroupTypes} from "./buildHTML";
+import {groupTypes as mathmlGroupTypes} from "./buildMathML";
+
+/* This file contains a list of functions that we parse, identified by
```

Comment:
"This file" is no longer correct.  It's now `functions.js`.

Incidentally, is there a particular reason to split `functions.js` into `functions.js` and `defineFunction.js`?  `import` should solve the awkward ordering issue automatically.  But maybe you prefer this conceptually.